labs/b1.md

```diff
@@ -101,12 +95,12 @@ verify your conjectures.
 
 3. What is the root directory in Linux filesystems?
 
-4. I want `ls` to do the following for me: Give me extra information such as
+4. `ls` has a lot of cool arguments. Try using them to get extra information such as
    file permissions, owner, group, size, and late date edited. In addition, I
    want to be able to see the size and have the files ordered by last date
    edited, with the oldest files on top. How would I do this?
 
-5. Instead of showing the first 10 lines of the file `big_data.txt`, I want head
+5. Instead of showing the first 10 lines of the file `big_data.txt`, I want `head`
    to show the first 4. How would I do that?
 
 6. What's the difference between `cat foo > out.txt` and `cat foo >> out.txt`?
```

Comment:
I'm wondering if Q6 should be moved to the next lab (where redirection is discussed)?

Reply:
I think it's fine, if anything we want to get people used to searching stuff up, even if it might be covered later.